dex-load/src/main/scala/com/wavesplatform/dex/load/TankGenerator.scala

```diff
@@ -11,9 +12,14 @@ import com.wavesplatform.wavesj.matcher.Order.Type
 import play.api.libs.json.{JsError, JsSuccess, JsValue}
 
 import scala.collection.JavaConversions.seqAsJavaList
+import scala.concurrent.duration.DurationInt
+import scala.concurrent._
 import scala.util.Random
 
 object TankGenerator {
+  val threadCount                                                = 5
+  private val executor: ExecutorService                          = Executors.newFixedThreadPool(threadCount)
```

Comment:
This thread pool creates user threads instead of daemon. So the application won't shutdown if we don't run `executor.shutdown` explicitly.

Consider either create `close` method, or use `ThreadFactory` to create daemon threads.

Reply:
added shutdown to finally of the main method